api.md

```diff
@@ -814,10 +814,11 @@ all attached hypertables.
 
 #### Optional Arguments [](delete_data_node-optional-arguments)
 
-| Name        | Description                                           |
-|-------------|-------------------------------------------------------|
-| `if_exists` | Prevent error if the data node does not exist. Defaults to false. |
-| `force`     | Force removal of data nodes from hypertables unless that would result in data loss.  Defaults to false. |
+| Name          | Description                                           |
+|---------------|-------------------------------------------------------|
+| `if_exists`   | Prevent error if the data node does not exist. Defaults to false. |
+| `force`       | Force removal of data nodes from hypertables unless that would result in data loss.  Defaults to false. |
+| `repartition` | Change the partitioning configuration so that all the remaining data nodes are used evenly. Defaults to true. |
```

Comment:
Do we want to add how the configuration is changed here? The wording suggests that it will not be even unless this option is provided, so why would a user ever want to have this to `false`?

Reply:
I suggest being more concrete on what this actually does:

`If set to true, makes the number of space partitions on all affected hypertables equal to the new number of data nodes (if such configuration exists). This ensures that the remaining data nodes are used evenly.`